ArchiveHunterDownloadManager/BulkOperations.m

```diff
@@ -100,7 +100,10 @@ + (BOOL) bulkForEach:(NSManagedObject *)bulk managedObjectContext:(NSManagedObje
 
 - (BOOL) prepareBulkEntries:(NSManagedObject *)bulk withError:(NSError **)err {
     return [BulkOperations bulkForEach:bulk managedObjectContext:_moc withError:err block:^(NSManagedObject *entry){
-        [self setupDownloadEntry:entry withBulk:bulk];
+        BulkOperationStatus itemStatus = [[entry valueForKey:@"status"] intValue];
+        if(itemStatus==BO_READY) {
+            [self setupDownloadEntry:entry withBulk:bulk];
+        }
```

Comment:
please put an else clause in here, and output an NSLog message giving the entry filename and status, and a message saying that it's not downloaded because the status is (whatever the status is)

Reply:
Done.